cves/kernel/CVE-2020-11609.yml

```diff
@@ -55,7 +55,12 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  By causing a NULL pointer dereference in the descriptor of the USB webcam
```

Comment:
Solid description, only questions would be what system/product was this found in? What is the stv06xx subsystem?